src/main/java/org/fundacionjala/at15/katas/langtonant/adhemar/Main.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.at15.katas.langtonant.adhemar;
+
+public class Main {
+    public static void main(String[] args) {
+        //Inputs
+        //Grid's size
+        int rows = 11;
+        int columns = 11;
+
+        int[] initialPosition = {5, 5};
+        int steps = 100;
+
+        //Create a grid
+        boolean color = false; //false: white, true: black
+        Grid grid = new Grid(rows, columns, color);
+
+        //Move the ant
+        Ant ant = new Ant(initialPosition[0], initialPosition[1]);
```

Comment:
What is the reason for using an array to send constructor arguments?

Reply:
Thinking about it there is no good reason, it would be easier to use normal variables, I'll change it